pkgs/top-level/all-packages.nix

```diff
@@ -39843,6 +39843,8 @@ with pkgs;
 
   qucs-s = qt6Packages.callPackage ../applications/science/electronics/qucs-s { };
 
+  simulide = callPackage ../applications/science/electronics/simulide { };
```

Comment:
For new packages, you need to add them to `pkgs/by-name` instead. Please check [here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md). It won't be necessary to add the new package to this file.

Reply:
I added my package to pkgs/by-name. It seems it is building fine by doing:

```bash
nix-build -A simulide
```

But i don't know why now the new syntax ``` nix run .#simulide``` and ```nix build #simulide``` stoped working. Is it some limitation of ```pkgs/by-name```?